go/vt/vtgate/gateway/discoverygateway.go

```diff
@@ -45,20 +46,20 @@ import (
 
 var (
 	cellsToWatch        = flag.String("cells_to_watch", "", "comma-separated list of cells for watching tablets")
-	tabletFilters       flagutil.StringListValue
 	refreshInterval     = flag.Duration("tablet_refresh_interval", 1*time.Minute, "tablet refresh interval")
 	refreshKnownTablets = flag.Bool("tablet_refresh_known_tablets", true, "tablet refresh reloads the tablet address/port map from topo in case it changes")
 	topoReadConcurrency = flag.Int("topo_read_concurrency", 32, "concurrent topo reads")
 	allowedTabletTypes  []topodatapb.TabletType
+	tabletFilters       flagutil.StringListValue
```

Comment:
Minor nit, this didn't actually change, right? If so can you please move it back above so it doesn't show up in diffs.

Reply:
heh, true enough; it felt more at home alphabetized with the other non-initialized flag; will move these back

✅